libs/vite-plugin-zephyr/src/lib/vite-plugin-zephyr.ts

```diff
@@ -29,7 +28,7 @@ export function withZephyr(_options?: VitePluginZephyrOptions): Plugin[] {
   const hooks = _options?.hooks;
   const plugins: Plugin[] = [];
   if (mfConfig) {
-    plugins.push(...federation(mfConfig));
+    plugins.push(...(federation(mfConfig) as Plugin[]));
```

Comment:
why this cast ?

Reply:
Probably type confusion
I need to check this closer.